src/app/util/af.h

```diff
@@ -467,125 +439,17 @@ void emberReverseMemCopy(uint8_t * dest, const uint8_t * src, uint16_t length);
  */
 EmberNodeId emberAfGetNodeId(void);
 
-#if defined(DOXYGEN_SHOULD_SKIP_THIS) || defined(EZSP_HOST)
-/**
- * @brief Generates a random key (link, network, or master).
- */
-EmberStatus emberAfGenerateRandomKey(EmberKeyData * result);
-#else
-#define emberAfGenerateRandomKey(result) emberGenerateRandomKey(result)
-#endif
-
-/**
- * @brief Returns the PAN ID of the local node.
- */
-EmberPanId emberAfGetPanId(void);
-
-/**
- * @brief Returns the radioChannel of the current network
- */
-uint8_t emberAfGetRadioChannel(void);
-
-/*
- * @brief Returns a binding index that matches the current incoming message, if
- * known.
- */
-uint8_t emberAfGetBindingIndex(void);
-
-/*
- * @brief Returns an address index that matches the current incoming message,
- * if known.
- */
-uint8_t emberAfGetAddressIndex(void);
-
 /**
  * @brief Returns the current network state.  This call caches the results
  *   on the host to prevent frequent EZSP transactions.
  */
 EmberNetworkStatus emberAfNetworkState(void);
 
-/**
- * @brief Returns the current network parameters.
- */
-EmberStatus emberAfGetNetworkParameters(EmberNodeType * nodeType, EmberNetworkParameters * parameters);
-
-/**
- * @brief Returns the current node type.
- */
-EmberStatus emberAfGetNodeType(EmberNodeType * nodeType);
```

Comment:
Is the EmberNodeType type unused after this?  Followup to remove it, please?

Reply:
Removed it and EmberChildData from type_stubs.h.